### DISCUSSION TITLE
Stripe app open-source?

### DISCUSSION THREAD

**Discussion by madmonkeyworks** [saleor/apps#960] [Q&A] [answered]
Hi there, 

I can't find Stripe listed under the Saleor apps, assuming it's not available as open-source. Is that right?
Is there a way we can get this app for our saleor instance running on our own infrastructure? Can we buy it?

Best wishes
Jan

**Comment by lkostrowski** [comment, marked as answer]
Hey

Stripe app is open source and available in the separate repository at the moment

https://github.com/saleor/saleor-app-payment-stripe

**Reply by madmonkeyworks** [reply]
Wonderful! 
Thank you!
